pom.xml

```
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>4.3</version>
+    <version>4.4</version>
```

Already has a higher value on the master branch.  Should merge the master branch to get the latest values.
```suggestion
    <version>4.6</version>
```

---

Thanks for noticing, I've done it.